src/WorkflowCore/Interface/IWorkflowController.cs

```diff
@@ -11,9 +12,11 @@ public interface IWorkflowController
         Task<string> StartWorkflow(string workflowId, int? version, object data = null, string reference=null);
         Task<string> StartWorkflow<TData>(string workflowId, TData data = null, string reference=null) where TData : class, new();
         Task<string> StartWorkflow<TData>(string workflowId, int? version, TData data = null, string reference=null) where TData : class, new();
-
+        Task<string> StartWorkflowForTest<TData>(string workflowId, WorkflowDefinition def, TData data = null, string reference = null)
```

Comment:
What is the purpose of this method?  Why do we need a different method when we are testing?